src/components/form-components.js

```diff
@@ -110,10 +110,21 @@ export class InputField extends Component {
         this.setState({ error: 'This field should be a JSON object. JSON objects begin with a { and end with a }' });
       }
     } else if (format === 'xpub') {
-      if (value.startsWith('xpub') && value.length === XPUB_LENGTH) {
-        this.setState({ error: null });
+      if (!coin) {
+        return;
+      }
+      if (coin.getFamily() !== 'xlm') {
```

Comment:
we have a `BaseCoin::isValidPub()` method since a couple of months ago, could you update this to use that method please?

Reply:
yup